pages/Static.vue

```diff
@@ -1,167 +1,151 @@
-        </h2>
-      </div>
-    </div>
-
-    <div class="container pt45 pb70">
-      <div class="row pl20 pt0">
-        <div class="col-sm-3">
-          <nav class="static-menu serif h4 mb35">
-            <ul class="m0 p0">
-              <li v-for="element in navigation" :key="element.id" class="mb10">
-                <router-link
-                  :to="localizedRoute(element.link)"
-                  class="cl-accent relative"
-                >
-                  {{ element.title }}
-                </router-link>
-              </li>
-            </ul>
-          </nav>
-        </div>
-        <div class="static-content h4 lh35 col-sm-9">
-          <component :is="activeComponent" />
-        </div>
-      </div>
-    </div>
+  <div id="static">
+    <SfBreadcrumbs
+      class="breadcrumbs desktop-only"
+      :breadcrumbs="breadcrumbs"
+    >
+      <template #link="{breadcrumb}">
+        <router-link :to="breadcrumb.route.link">
+          {{ breadcrumb.text }}
+        </router-link>
+      </template>
+    </SfBreadcrumbs>
+    <SfContentPages
+      :active="currentRoute.title"
+      :title="currentRoute.title"
+      @click:change="updateRoute"
+    >
+      <SfContentPage v-for="n in navigation" :key="n.title" :title="n.title">
+        <AStatic :content="content" />
+      </SfContentPage>
+    </SfContentPages>
   </div>
 </template>
 
 <script>
 import i18n from '@vue-storefront/i18n';
-import Breadcrumbs from 'theme/components/core/Breadcrumbs';
-import StaticExample from 'theme/components/theme/blocks/Static/Example';
-import StaticShortExample from 'theme/components/theme/blocks/Static/Short';
+import {
+  SfBreadcrumbs,
+  SfContentPages
+} from '@storefront-ui/vue';
 import { getPathForStaticPage } from 'theme/helpers';
-import { localizedRoute } from '@vue-storefront/core/lib/multistore';
+import CmsPage from '@vue-storefront/core/pages/CmsPage';
+import AStatic from 'theme/components/atoms/a-static';
 
 export default {
+  name: 'Static',
   components: {
-    Breadcrumbs
-  },
-  metaInfo () {
-    return {
-      title: this.$route.meta.title || this.$props.title,
-      meta: this.$route.meta.description
-        ? [{ vmid: 'description', description: this.$route.meta.description }]
-        : []
-    };
-  },
-  props: {
-    title: {
-      type: String,
-      required: true
-    },
-    page: {
-      type: String,
-      required: true
-    }
+    SfBreadcrumbs,
+    SfContentPages,
+    AStatic
   },
+  mixins: [CmsPage],
   data () {
     return {
       navigation: [
+        { title: i18n.t('About us (Magento CMS)'), link: getPathForStaticPage('/about-us'), isCms: true },
+        { title: i18n.t('Customer service (Magento CMS)'), link: getPathForStaticPage('/customer-service'), isCms: true },
+        { title: i18n.t('Customer service'), link: '/customer-service' },
+        { title: i18n.t('Legal notice'), link: '/legal' },
+        { title: i18n.t('Store locator'), link: '/store-locator' },
```

Comment:
previously it was `localizedRoute('/store-locator')`. It's not needed anymore?

Reply:
`localizedRoute('/store-locator')` produces `/store-locator` path and also I don't see any other `store-locator` paths in the router (I mean other than `/store-locator` - to handle it in different way for non-default store configuration), so I'm guessing it is not needed anymore 🤔

Reply:
good point